.eslintrc

```
@@ -0,0 +1,8 @@
+{
+  "extends": [
+    "airbnb-base"
+  ],
+  "rules": {
+    "no-underscore-dangle": 0
+  }
```

Можно лучше: выключать правила `eslint` точечно.
Сейчас правило полностью блокируется. Это плохо, потому что возможно именование переменных с нижним подчеркиванием везде, даже там, где без этого можно обойтись. Лучше так:
```suggestion
  "rules": {
    "no-underscore-dangle": ["error", { "allow": ["_id"] }]
  }
```